diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index af8661d..5cde1cc 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -4,28 +4,28 @@ ifeq ($(CONFIG_MACH_XIAOMI),y)
 ifeq ($(CONFIG_MACH_XIAOMI_SM8150),y)
 ifeq ($(CONFIG_BUILD_ARM64_DT_OVERLAY),y)
 	dtbo-$(CONFIG_ARCH_SM8150) += \
-		andromeda-sm8150-overlay.dtbo \
-		andromeda-sm8150-p1-overlay.dtbo \
-		andromeda-sm8150-p1_1-overlay.dtbo \
-		andromeda-sm8150-p2-overlay.dtbo \
-		andromeda-sm8150-p2_1-overlay.dtbo \
-		andromeda-sm8150-p2_2-overlay.dtbo \
-		andromeda-sm8150-p2_21-overlay.dtbo \
-		andromeda-sm8150-p2_22-overlay.dtbo \
+#		andromeda-sm8150-overlay.dtbo \
+#		andromeda-sm8150-p1-overlay.dtbo \
+#		andromeda-sm8150-p1_1-overlay.dtbo \
+#		andromeda-sm8150-p2-overlay.dtbo \
+#		andromeda-sm8150-p2_1-overlay.dtbo \
+#		andromeda-sm8150-p2_2-overlay.dtbo \
+#		andromeda-sm8150-p2_21-overlay.dtbo \
+#		andromeda-sm8150-p2_22-overlay.dtbo \
 		cepheus-sm8150-overlay.dtbo \
 		crux-sm8150-overlay.dtbo \
 		nabu-sm8150-overlay.dtbo \
 		raphael-sm8150-overlay.dtbo \
 		vayu-sm8150-overlay.dtbo
 
-andromeda-sm8150-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
-andromeda-sm8150-p1-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
-andromeda-sm8150-p1_1-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
-andromeda-sm8150-p2-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
-andromeda-sm8150-p2_1-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
-andromeda-sm8150-p2_2-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
-andromeda-sm8150-p2_21-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
-andromeda-sm8150-p2_22-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
+#andromeda-sm8150-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
+#andromeda-sm8150-p1-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
+#andromeda-sm8150-p1_1-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
+#andromeda-sm8150-p2-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
+#andromeda-sm8150-p2_1-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
+#andromeda-sm8150-p2_2-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
+#andromeda-sm8150-p2_21-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
+#andromeda-sm8150-p2_22-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
 cepheus-sm8150-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
 crux-sm8150-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
 nabu-sm8150-overlay.dtbo-base := sm8150-xiaomi.dtb sm8150-v2-xiaomi.dtb sm8150p-xiaomi.dtb sm8150p-v2-xiaomi.dtb
